ansible/README.md

```diff
@@ -15,36 +14,86 @@ You will need your public key deployed to the host for ssh access
 
 You may need to connect to the host once with ssh to make sure it's in your list of known hosts
 
+If you install [casey/just](https://github.com/casey/just), then some tasks
+will be simplified.
+
 ## Before Running
 
-Create secret files under secrets-[env]:
+**Setup the Ansible Vault Passphrase**
+There are passwords, secret keys, and other sensitive information required for everything
+to work. These secrets are included in the repo and they are encrypted using
+ansible-vault.  To run ansible-successfully, you will need the passphrase to
+decrypt them.  Ask a team member. Then, run:
+
+```console
+$ echo -n "passphrase" >> .passworf
+```
 
-* 1nfinite.pem (The pem obtained from auth0)
-* web-portal.env (env file with the required environment values for the web-portal service)
-* api.env (env file with the required environment values for the api service)
+*Alternatively:*
+
+```console
+$ just cache-pass
+```
 
 ## Running
 
-`ansible-playbook ./base_playbook.yml`
+### Common Task: Site Deployment
+
+First, deploy our code: `ansible-playbook -l staging deploy_site_playbook.yml`
+* alternative: `just deploy staging`
+
+Next, restart containers.
 
-## First Time Setup
+```
+$ ssh infinite@infinite.industries
+
+prod $ cd docker-files
+prod $ docker-compose up -d
+prod $ sudo systemctl restart nginx
+```
+
+### Task: First Time Setup
 
 **These steps only needs to happen once**
 
-After running ansible you will need to ssh into the machine and run:
+1. Add the IP address and other info the appropriate section of the `hosts`
+   file.  These instructions assume a new host is being added to the staging
+   environment.
+
+2. Do the initial install: `ansible-playbook -l staging base_playbook.yml`
+* alternative: `just init staging`
+
+3. Setup certbot (per these [instructions](https://certbot.eff.org/instructions?ws=nginx&os=ubuntufocal).
+
+```
+$ sudo certbot certonly --nginx
+```
+* enter: *staging.infinite.industries,staging-api.infinite.industries* (or *infinite.industries,api.infinite.industries* if this is for prod)>
```

Comment:
Is the * bookending this here to make it bold, or is it a wild card? I'm a little worried I might copy that if tying to run while not very awake.

Reply:
Might be worth keeping the link to https://certbot.eff.org/instructions?ws=nginx&os=ubuntufocal somewhere around. There's nothing more fun than having to go back to google and try to guess at where you learned to do something while you're sleepy and prod is down